scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -1497,16 +1498,24 @@ function Remove-IotEdgeContainers {
 }
 
 function Get-DockerCommandPrefix {
+    $prefix = ""
+    # in case the installation has not been completed
+    if (-not (Get-Command "docker.exe" -ErrorAction SilentlyContinue)) {
+        $prefix = "$MobyInstallDirectory\"
+    }
+
     switch ($ContainerOs) {
         'Linux' {
-            return '"docker"'
+            return ('"{0}docker"' -f $prefix)
```

Comment:
Shouldn't do this for Linux. We want to use Docker for Windows' `docker.exe` in that case.

Reply:
Fixed.